vertical-pod-autoscaler/pkg/utils/vpa/api.go

```diff
@@ -49,14 +49,14 @@ type patchRecord struct {
 	Value interface{} `json:"value"`
 }
 
-func patchVpa(vpaClient vpa_api.VerticalPodAutoscalerInterface, vpaName string, patches []patchRecord) (result *vpa_types.VerticalPodAutoscaler, err error) {
+func patchVpaStatus(vpaClient vpa_api.VerticalPodAutoscalerInterface, vpaName string, patches []patchRecord) (result *vpa_types.VerticalPodAutoscaler, err error) {
```

Comment:
Changes in this file look like they're part of #5766 too

Reply:
I kept two commits here: 1 is a revert for a commit that somehow fell into this branch, and another for all of my changes.  I'll try rebasing and see if I can skip the revert commit I added.